Gemfile

```diff
@@ -48,3 +65,7 @@ group :development do
   gem 'spring'
 end
 
+group :test do
+  gem "capybara", "~> 2.7"
+  gem "launchy"
+end
```

Comment:
Versions need to be properly filled as well, otherwise, as it's the case on `platform` it's gonna be impossible to properly update the gems with `bundle update`